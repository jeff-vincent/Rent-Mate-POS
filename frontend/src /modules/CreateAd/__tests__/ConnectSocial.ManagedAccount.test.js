import React from 'react';
import { shallow } from 'enzyme';
import { 
    ManagedAccounts,
    managedStateReducer,
    ManagedAccountStatusComponent,
    actionStates
} from '../components/ConnectSocial.ManagedAccount';

const title = 'Test Title';
let token = localStorage.getItem('token')

 describe('managedStateReducer should return proper state on correct action', () => {
    it('it should throw error on empty parameters', () => {
        expect(managedStateReducer({}, {})).ToThrow(Error);
    });
    it('it should change business_info state to isLoading to true', () => {
        expect(managedStateReducer({},{
            type: actionStates.USER_PROFILE_LOADING_STATE
        })).toBeDefined()
    })
 })

 