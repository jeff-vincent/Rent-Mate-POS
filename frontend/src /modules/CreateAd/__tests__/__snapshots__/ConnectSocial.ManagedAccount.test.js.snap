// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render the Title Component correctly 1`] = `
<div>
  <div>
    <WithStyles(ForwardRef(FormControlLabel))
      control={
        <ForwardRef(WithStyles)
          checked={false}
          color="primary"
          name="Managed Account"
          onChange={[Function]}
        />
      }
      label="Make Managed Account"
    />
  </div>
  <WithStyles(ForwardRef(Button))
    color="primary"
    onClick={[Function]}
    variant="outlined"
  >
    What is a Managed Account?
  </WithStyles(ForwardRef(Button))>
  <WithStyles(ForwardRef(Dialog))
    TransitionComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "render": [Function],
      }
    }
    aria-describedby="alert-dialog-slide-description"
    aria-labelledby="alert-dialog-slide-title"
    keepMounted={true}
    onClose={[Function]}
    open={false}
  >
    <WithStyles(ForwardRef(DialogTitle))
      id="alert-dialog-slide-title"
    >
      What is a Managed Account?
    </WithStyles(ForwardRef(DialogTitle))>
    <WithStyles(ForwardRef(DialogContent))>
      <WithStyles(ForwardRef(DialogContentText))
        id="alert-dialog-slide-description"
      >
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Unde rerum, consequuntur adipisci, laboriosam sit ullam saepe illum fugiat ea explicabo nostrum impedit earum vel eum veniam fugit omnis hic voluptate.
      </WithStyles(ForwardRef(DialogContentText))>
    </WithStyles(ForwardRef(DialogContent))>
    <WithStyles(ForwardRef(DialogActions))>
      <WithStyles(ForwardRef(Button))
        color="primary"
        onClick={[Function]}
      >
        Ok
      </WithStyles(ForwardRef(Button))>
    </WithStyles(ForwardRef(DialogActions))>
  </WithStyles(ForwardRef(Dialog))>
</div>
`;
